code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -102,7 +102,11 @@
 	. = ..()
 	if(!ethereal_light)
 		return
-
+	if(default_color != H.dna.features["ethcolor"])
+		var/new_color = H.dna.features["ethcolor"] //can I just replace the default color here?
```

Comment:
Leaving a draft due to this. Can I just set the default to equal the new color, or should I keep this as is (or do it a different way?)

Reply:
This was discussed on discord. Default color isn't actually used anywhere, so the way I did it is good.